t/data/dbic_fixtures/300-Run.yml

```
@@ -45,6 +45,19 @@
   folder_name: 123456_IL2_1234_A_80B5TABXX 
   folder_path_glob: /{export,nfs}/sf33/ILorHSany_sf33/*/
   team: '"RAD"'
+- actual_cycle_count: 57
+  batch_id: 55
+  expected_cycle_count: 66
+  flowcell_id: dummy_flowcell1
+  id_instrument: 48
+  id_instrument_format: 7
+  id_run: 1235
+  id_run_pair: 0
+  is_paired: 0
+  priority: 1
+  folder_name: 123456_IL2_1234_A_80B5TABXX 
```

Collisions with folder names is a bad idea because we still use folder names to get run ids in some cases. It may not cause problems now but may cause unexpected results for other people updating tests in the future.

```suggestion
  folder_name: 123457_IL2_1235_A_81B5TABXX 
```